lib/builders/gpu-builder.ts

```diff
@@ -158,7 +158,7 @@ function addGpuNodeGroup(options: GpuOptions): clusterproviders.ManagedNodeGroup
     return {
         id: "mng-linux-gpu",
         amiType: NodegroupAmiType.AL2_X86_64_GPU,
```

Comment:
What about `NodegroupAmiType` ? Also please remove the blocksize field as discussed yesterday

Reply:
I think NodegroupAMIType is better off being fixed. removing it altogether as a GPU Option is the way to go

Reply:
Agree, also remove the blockdevice one